static/stylesheets/application.css

```
@@ -1040,10 +1040,9 @@ h6 + .id-link-button {
 .article ol>li {
     padding: 5px 0 5px 10px;
     margin: 0 0 15px 30px;
```

左側に大きく開く余白が気になりました。
Garoonと同じくらい空いてるんだけど、kintone/Slashは右側にメニューがあって記事を掲載する部分が狭いのもあるので、狭めてもいいのかもと思いました。
```suggestion
    margin: 0 0 15px 0px;
```